### ISSUE TITLE
Pyro server can't start and frida OK

### ISSUE THREAD

**Issue by janemba** [federicodotta/Brida#10]
Hi,

I tried to make Brida start but I got the following error :

`
Exception starting Pyro server java.util.concurrent.TimeoutException java.util.concurrent.FutureTask.get(FutureTask.java:205) burp.BurpExtender.launchPyroServer(BurpExtender.java:914) burp.BurpExtender.actionPerformed(BurpExtender.java:1367) javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:2022) javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2348) javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:402) javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:259) javax.swing.plaf.basic.BasicButtonListener.mouseReleased(BasicButtonListener.java:252) java.awt.Component.processMouseEvent(Component.java:6533) javax.swing.JComponent.processMouseEvent(JComponent.java:3324) java.awt.Component.processEvent(Component.java:6298) java.awt.Container.processEvent(Container.java:2236) java.awt.Component.dispatchEventImpl(Component.java:4889) java.awt.Container.dispatchEventImpl(Container.java:2294) java.awt.Component.dispatchEvent(Component.java:4711) java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4888) java.awt.LightweightDispatcher.processMouseEvent(Container.java:4525) java.awt.LightweightDispatcher.dispatchEvent(Container.java:4466) java.awt.Container.dispatchEventImpl(Container.java:2280) java.awt.Window.dispatchEventImpl(Window.java:2746) java.awt.Component.dispatchEvent(Component.java:4711) java.awt.EventQueue.dispatchEventImpl(EventQueue.java:758) java.awt.EventQueue.access$500(EventQueue.java:97) java.awt.EventQueue$3.run(EventQueue.java:709) java.awt.EventQueue$3.run(EventQueue.java:703) java.security.AccessController.doPrivileged(Native Method) java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80) java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:90) java.awt.EventQueue$4.run(EventQueue.java:731) java.awt.EventQueue$4.run(EventQueue.java:729) java.security.AccessController.doPrivileged(Native Method) java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:80) java.awt.EventQueue.dispatchEvent(EventQueue.java:728) java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201) java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116) java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105) java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101) java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93) java.awt.EventDispatchThread.run(EventDispatchThread.java:82)  
`

Frida is working fine (I use it with USB) I got the process list. Also, nothing is running on port 9999.

I'm using python 2.7 with frida version 11.0.0 and pyro4 version 4.71.

Do you know how can i fix this issue ?

**Comment by janemba** [comment]
and I'm using brida version 0.2.

**Comment by janemba** [comment]
Well, how  Brida start Pyro ?

**Comment by federicodotta** [comment]
Hi,

Pyro is started with a system exec, using python 2.

Have you configured correctly all the paths in the Configuration tab? Python path, Frida JS file, etc.
The Application ID is the correct one? If you try to use Frida to spawn your application from the console, does it work?
Are you using Python 2? You can't use Python 3.

Federico

**Comment by federicodotta** [comment]
Hi Chan9390,

yes, Brida works also on Burp Suite Community edition. Usually that kind of issue is caused by an error in the configurations or by a missing dependency (Pyro4, python 2, etc.)

Federico

**Comment by federicodotta** [comment]
Have you started frida-server on your mobile device? Have you configured properly the mode of connection with frida-server? (local or remote)

If yes, you can run directly Frida with the following command (with -R instead of -U if you are using remote connection with frida-server):
`frida -U -f #APP_ID -l #JS_FRIDA_FILE --no-pause`

**Comment by janemba** [comment]
@federicodotta 

Yes frida-server server is started on the device and work properly. I use "Frida Local" for connection mode (with USB).

So I run the command manually and I got this error :
```
Failed to spawn: unable to handle 64-bit processes due to build configuration
```

Hmmm...

**Comment by federicodotta** [comment]
Maybe you are using wrong Frida binaries :-)

Anyway, it is not an issue related to Brida.  Can I close the issue?

Federico

**Comment by janemba** [comment]
Hi @federicodotta,

Well that's the problem, there is not so much stuff to check and I already like 10 thousands times. Also, I look at source code to get an idea of what you are doing and as you see on my previous post I did the same command that Brida is doing and it works. So, what I should check exactly ?

**Comment by invisible0** [comment]
hi @janemba 
i had the same issue, i tried to use frida-server-arm64. and now it's work correctly.
